.github/ansible/production/deploy.yml

```diff
@@ -0,0 +1,68 @@
+- name: Deploy Nest to Production
+  hosts: nest_production
+  tasks:
+    - name: Copy Nginx configuration file
+      ansible.builtin.copy:
+        src: '{{ github_workspace }}/nginx/staging.conf'
```

Comment:
We want a separate config.

Reply:
Sure ...will be creating production.conf

Reply:
@arkid15r I have introduced production.conf now ..let me know if all the changes are good to go.
Thanks !